curriculum/challenges/english/25-front-end-development/quiz-css-colors/66ed8fe1f45ce3ece4053eb1.md

```diff
@@ -484,3 +484,325 @@ background: linear-gradient(blue, red)
 ```css
 background: linear-gradient(red, blue)
 ```
+
+## --quiz--
+### --question--
+
+#### --text--
+
+What are the 3 secondary colors in Color Theory?
+#### --distractors--
+`Blue`, `Orange`, `Yellow`
+
+---
+`Purple`, `Green`, `Yellow`
+
+---
+`Green`, `Orange`, `Pink`
+#### --answer--
+`Green`, `Orange`, `Purple`
+
+### --question--
+
+#### --text--
+
+What feelings do warm colors evoke?
+#### --distractors--
+Subtlety, Isolation, Loneliness
+
+---
+Clarity, Freshness, Cleanliness
+
+---
+Melancholy, Nonchalance, Frostiness
+#### --answer--
+Comfort, Warmth, Coziness
+
+### --question--
+
+#### --text--
+Which type of color scheme uses colors that are equidistant from each other on the color wheel?
+
+#### --distractors--
+`Complementary`
```

Comment:
also no backticks here, things in backticks can't be translated, this doesn't make sense to not translate